client/my-sites/email/email-management/titan-manage-mailboxes/index.jsx

```diff
@@ -103,7 +103,7 @@ class TitanManageMailboxes extends Component {
 				{ selectedSite && hasSubscription && <QuerySitePurchases siteId={ selectedSite.ID } /> }
 
 				<Main wideLayout={ true }>
-					<DocumentHead title={ translate( 'Manage all mailboxes' ) } />
+					<DocumentHead title={ translate( 'Manage All Mailboxes' ) } />
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.